Answers.md

```diff
@@ -1,5 +1,16 @@
 1.  Explain the differences between `client-side routing` and `server-side routing`.
-1.  What does HTTP stand for?
-1.  What does CRUD stand for?
-1.  Which HTTP methods can be mapped to the CRUD acronym that we use when interfacing with APIs/Servers.
-1.  Mention three tools we can use to make AJAX requests
\ No newline at end of file
+    A client-side routing don't cause whole page to refresh, but server-side routing does.
+
+2.  What does HTTP stand for?
+    Hypertext transfer protocol
+
+3.  What does CRUD stand for?
+    Create, Read, Update, Delete
+
+4.  Which HTTP methods can be mapped to the CRUD acronym that we use when interfacing with APIs/Servers.
+    we use put method to mapped the Servies
+
+5.  Mention three tools we can use to make AJAX requests
+    Axios Library
```

Comment:
.then.catch are Promise methods, not Ajax requests specifically. Jquery is another way to make AJAX requests.